Lib/abc.py

```diff
@@ -121,6 +121,38 @@ def _abc_caches_clear(cls):
             """Clear the caches (for debugging or testing)."""
             _reset_caches(cls)
 
+def update_abstractmethods(cls):
+    """Repopulate the abstract methods of an abstract class, or a subclass of
+    an abstract class.
+    
+    If a class has had one of its abstract methods implemented after the
+    class was created, the method will not be considered implemented until
+    this function is called. Alternativly, if a new abstract method has been
+    added to the class, it will not be considered to require that method
+    implemented until this function is called.
```

Comment:
I'm not sure what this is trying to say: "will not be considered to require that method implemented". Could you reword it?